nixos/modules/system/boot/loader/defaults.nix

```
@@ -0,0 +1,35 @@
+{ config, lib, utils, ... }:
+
+with lib;
+
+{
+  meta = {
+    maintainers = with maintainers; [ raitobezarius ];
+  };
+
+  options.boot.loader.defaults = utils.mkLoaderOption {
+    enable = mkOption {
+      # make me readonly
+      default = false;
+      type = types.bool;
+      internal = true;
+    };
+
+    timeout = mkOption {
+      default = 5;
+      type = types.nullOr types.int;
+      description = lib.mdDoc ''
+        Timeout (in seconds) until loader boots the default menu item. Use null if the loader menu should be displayed indefinitely.
+
+        Not all bootloaders supports timeouts.
```

```suggestion
        :::NOTE
        Not all bootloaders supports timeouts.
        :::
```
Was that the syntax? Can't remember and searching on mobile is not fun.

---

Let me know when you find the syntax in the meantime, will resolve until then.